notebooks/hybrid_retrieval_bm42.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
```

Comment:
<p>This section is missing a line that tells you something like: Let's take a look at various retrieval techniques like.. Or something. Splade comes as a surprise here. A line to intro to the subsections would be useful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
improved